### PR TITLE
Django init

### PR DESCRIPTION
**Pull request by Dyanni-Bigham**
Created a bare bones Django project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # codeback
 Codevaak's code back.
+
+## Setup
```
Comment (Dyanni-Bigham) [inline README.md:4]: Is this okay for a setup?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+1. install python3
+2. create a virtual environment
+3. pip install -m Django
+4. use django-admin startproject to create the project
```
Comment (wongcoder) [inline README.md:8]: Where's the FAQ? Did you not run into any big issues?

**Comment by wongcoder** [comment]
Please rebase your code into one commit before submitting the PR.

**Review by wongcoder — CHANGES_REQUESTED** [review]

**Review by Dyanni-Bigham — COMMENTED** [review]
I didn't run into technical issues.

File: README.md (hunk 1 above)
Comment (wongcoder) [reply README.md:4]: lgtm

**Pull request merged**